applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking.yaml

```diff
@@ -21,18 +21,48 @@ extensions:
   - libgxf_serialization.so
   - libgxf_stream_playback.so
   - gxf_extensions/lstm_tensor_rt_inference/libgxf_lstm_tensor_rt_inference.so
+  - gxf_extensions/deltacast_videomaster/libgxf_videomaster.so
```

Comment:
@jjomier if user builds without using `--with deltacast_videomaster`, then it looks like we'll get some additional warnings when application starts:

 [warning] Unable to load extension from 'gxf_extensions/deltacast_videomaster/libgxf_videomaster.so' (error: gxf_extensions/deltacast_videomaster/libgxf_videomaster.so: cannot open shared object file: No such file or directory)

Reply:
That's the drawback of combining the applications. That said, you are right, let's comment this line out and document that it should be added when using Deltacast. I'll push a fix.